source/blog/2016-04-27-baseURL.md

```diff
@@ -0,0 +1,104 @@
+
+#### Implicit Behavior
+
+Not only did we have two properties, if only one of `baseURL` or `rootURL` was set Ember CLI tried to guess what the other should be set to based upon the one we did have. This implicit behavior violated the principle of least surprise for many people. If we had always required both to be specified and provided very clear documentation about what each did we may have averted some of the pain.
+
+#### Integration With Other Applications
+
+For example, not every Ember application is full green field development. If you happened to have legacy user-generated content or other data which had relative links, this didn't play nicely with the `<base>` tag. Rewriting this content is an option, but incurs cost and bugs we can more-simply avoid by not including the `<base>` tag.
+
+#### SVG
+
+And worst of all, in something that we can't code around, SVG has `<use>` cases that require addressing content fragments which break terribly with the `<base>` tag. As long as you didn't use SVG all of the other reasons were manageable, but it turns out that `<use xlink:xhref="#__SVG_ID__"></use>` doesn't play nicely with the `<base>` tag. We like to play nicely with the web ecosystem and breaking weird things in SVG is the final straw that pushed us over the edge on removing the `<base>` tag.
+
+### How Do I Migrate?
+
+This depends primarily on how your application is deployed. It's our goal is to make it easy for you to achieve the same behaviors you've had in the past. In an uncustomized Ember CLI application you will likely not encounter any issue. We've been very careful to make sure that the out of the box experience is as smooth as possible.
+
+From this point forward in `config/environment.js` you should simply set the `rootURL` property on the `ENV` object to where your application will appear on the server:
+
+```javascript
+module.exports = function(environment) {
+  var ENV = {
+    /* ... */
+    rootURL: '/path/to/application/'
+    /* ... */
+  };
+
+  return ENV;
+}
+```
+
+#### New Development
+
+Your development environment will serve all of its assets using the bundled `ember server` command. This should be unaffected as all assets are still inside of the Ember CLI world.
+
+#### Upgrading
+
+If you are upgrading your application to the new version of Ember CLI you should pay close attention to the configuration changes inside of [the](https://github.com/ember-cli/ember-cli/pull/5792/files#diff-42e917b9304df0e99048f335019647bc) [modified](https://github.com/ember-cli/ember-cli/pull/5792/files#diff-f4d09ffc13efe496dd246031d423e166) [blueprint](https://github.com/ember-cli/ember-cli/pull/5792/files#diff-9588ab8abb99afa6cac30dd0e2b86041) [files](https://github.com/ember-cli/ember-cli/pull/5792/files#diff-3b0f91e0e9f317eecebfad38da2f9b4c). We have also inserted deprecation warnings which will hopefully guide you through each of these changes.
```

Comment:
I didn't actually care to make it visually distinct since my goal was to encourage people to look at "the modified blueprint files." Each of these links is just an anchor link inside of the same page so scrolling around once landed should be fine. For assistive tech users the separate links will be available, so it will be equivalently accessible.